sdk/core/Azure.Core/tests/TransportFunctionalTests.cs

```diff
@@ -407,13 +412,19 @@ public async Task CanGetAndSetMultiValueResponseHeaders(string headerName, strin
             CollectionAssert.Contains(response.Headers, new HttpHeader(headerName, joinedHeaderValues));
         }
 
-        [TestCaseSource(nameof(HeadersWithValuesAndType))]
-        public async Task CanRemoveHeaders(string headerName, string headerValue, bool contentHeader)
+        [TestCaseSource(nameof(HeadersWithValues))]
+        public async Task CanRemoveHeaders(string headerName, string headerValue, bool contentHeader, bool supportsMultiple)
         {
+            // Some headers are required
+            bool checkOnServer = headerName != "Content-Length" && headerName != "Host";
```

Comment:
Use the const `HttpRequestHeader` values here?

Reply:
We use literal names in this test file. I'll keep it for consistency.